docs/developer-docs/integrations/internet-identity/integrate-identity.md

```diff
@@ -1,120 +1,237 @@
 # Internet Identity integration
 
 ## Overview
-This guide shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
+This guide shows an example of to integrate Internet Identity into an application by using a simple 'Who am I?' backend canister and a frontend UI that returns the Internet Identity principal of the user who calls the backend's `whoami` method. 
 
-This is a standalone project that you can copy to your own project.
+This project uses the **pullable** version of the Internet identity canister. A pullable canister is a canister that provides a public service at a static canister ID. To learn more about pullable canisters, please see the documentation [here](/docs/current/developer-docs/setup/pulling-canister-dependencies). 
 
+### Prerequisites
 
-* [`dfx`](https://github.com/dfinity/sdk/releases/latest) version 0.10.0 or later
-* Rustup with target `wasm32-unknown-unknown` (see [rustup instructions](https://rust-lang.github.io/rustup/cross-compilation.html)).
-* CMake
-* [`ic-wasm`](https://github.com/dfinity/ic-wasm).
-* Node.js v16+
+Before you start, verify that you have:
 
-## Usage
+- Downloaded and installed [`dfx`](https://github.com/dfinity/sdk/releases/latest) version 0.10.0 or later.
+- [Node.js v16+.](https://nodejs.org/en).
 
-The following commands will start a replica, install the development Internet Identity canister, and run the test suite:
+### Step 1: To get started, open a terminal window and clone the sample project's repository: 
 
-```bash
-git clone https://github.com/dfinity/internet-identity.git
-cd internet-identity/demos/using-dev-build
-dfx start --background --clean
-npm ci
-dfx deploy --no-wallet
+```
+dfx start --clean --background
+git clone https://github.com/krpeacock/auth-client-demo.git
```

Comment:
i was hoping for an example that starts with a clean new project, like the ones where we show people how to integrate with ICRC1 or ICP ledger canisters. 
i shared the steps on how this could be achieved [here](https://dfinity.slack.com/archives/C01TTU2B15Z/p1695830602138719?thread_ts=1695751895.902199&cid=C01TTU2B15Z)

this way we keep things as simple as possible, we could of course link to more elaborate examples at the end of the document for further reading.

is there a particular reason kyles demo was picked here instead?

Reply:
I've revised it to use the example repo you created instead. Please let me know if this is more aligned with what you had in mind.